### PR TITLE
Library improvements

### PR DESCRIPTION
**Pull request by msal4**
Fixes some issues mentioned in #1904
- Resize library items and show more.
- Show library menu on mobile
- Resize icons on mobile

### CODE REVIEW

**Comment by dwelle** [comment]
- Not sure whether the library items aren't too small. Would probably need real-world use cases, I myself haven't used it that much yet.

Few notes on mobile (I'm on Android): 

- the `+` icon in the shape library doesn't show over the shape most of the time (it shows once I tap/hold finger over the shape).
- nor does the `x` icon, and it's easy to misclick and mistakenly remove an element from the library.

**Comment by msal4** [comment]
I haven't looked into the library + and x icons, I'll fix them right away.
As for the library items, in my opinion, they are small but I don't think they are too small. It also wouldn't hurt if we made them a little bigger.

**Comment by msal4** [comment]
I made the icons visible on mobile without the need for hovering. I think it's very usable now.

File: src/components/LibraryUnit.scss [hunk 1]
```diff
@@ -73,3 +73,12 @@
...
 }
+
+// @media (max-width: 425px) {
+//   .library-unit__removeFromLibrary,
+//   .library-unit__removeFromLibrary:hover,
+//   .library-unit__removeFromLibrary:active {
+//     width: 16px;
+//     height: 16px;
+//   }
+// }
```
Comment (dwelle) [inline src/components/LibraryUnit.scss:84]:
```suggestion
```

Comment (dwelle) [reply src/components/LibraryUnit.scss:84]: Otherwise looking good.

Comment (msal4) [reply src/components/LibraryUnit.scss:84]: I forgot to delete this.

**Comment by msal4** [comment]
These changes address issue #1901 mentioned in #1904.
It adds the ability to export and import libraries. <br>
illustration[img]
<br>
The exported file has the extension `.excalidrawlib`. Could be `.json` or any other extension. It probably needs a dedicated button on the side so it's clear how to import a library.

**Comment by dwelle** [comment]
I'd maybe leave exporting/importing for another PR, but here are my notes:

- it should export whole library, not just a single item
- as such, there should be an explicit button for export/import inside the library dialog, not via contextmenu on an item
- the export file dialog doesn't show correct file extension, at lest on windows and using the new FS. Are you testing on chrome or Safari/FF?

**Comment by msal4** [comment]
I'm using a mac and I tested on Firefox and Chrome. It shows the correct file extension ".excalidrawlib".
Should I open another PR?

**Comment by dwelle** [comment]
> I'm using a mac and I tested on Firefox and Chrome. It shows the correct file extension ".excalidrawlib". [EDIT] I just checked it on windows, it shows the same extension as well.

:shrug:

I checked how we do it elsewhere, and we're supposed to pass `extensions` property. It works for me that way.

```
fileSave(blob, { fileName: "library", extensions: ["excalidrawlib"] });
```

> Should I open another PR?

Prob best so we can discuss & review it separately, and it won't block this PR.

**Comment by msal4** [comment]
Great, will do. Thanks!

**Comment by dwelle** [comment]
I'm gonna merge this since it doesn't seem anyone will chime in any time soon. We can tweak the design later if need be. Thanks for the PR!

**Comment by msal4** [comment]
My pleasure

**Pull request merged**